projects/developer/src/app/common/components/queue-load/component.ts

```diff
@@ -51,17 +62,48 @@ export class QueueLoadComponent implements OnInit, OnDestroy, OnChanges {
     };
     constructor(
         private messageService: MessageService,
-        private queueApiService: QueueApiService
+        private queueApiService: QueueApiService,
+        private themeService: ThemeService
     ) {
     }
 
+    private updateText() {
```

Comment:
I think this could be simplified a bit:
```ts
    const updateChartColors = (themeName: string) => {
        const color = themeName === 'light' ? ColorService.FONT_LIGHT_THEME : ColorService.FONT_DARK_THEME;
        
        this.options.legend.labels.fontColor = color;
        this.options.scales.yAxes[0].ticks.fontColor = color;
        this.options.scales.xAxes[0].ticks.fontColor = color;

        setTimeout(() => {
            this.chart.reinit();
        });
    };
    
    updateChartColors(this.themeService.getActiveTheme().name);
    
    this.themeSubscription = this.themeService.themeChange.subscribe(theme => {
        updateChartColors(theme.name);
    );
```

Also I don't think the setTimeout actually needs a time value, it will run outside of the Angular loop anyways. Try it though.

Since this is a bit smaller, you could just put this directly into the `ngOnInit` if you'd like.